src/components/popup/details/details.jsx

```diff
@@ -28,24 +28,28 @@ export default class Details extends Component {
 		return Object.values(vendors);
 	};
 
-	filterVendors = ({ isCustom = null, purposeIds = [], featureIds = [] } = {}) => {
-		return this.getVendors().filter(vendor => {
-			if (isCustom !== null && (isCustom && vendor.globalId || !isCustom && !vendor.globalId)) {
-				return false;
+	filterVendors = ({ isCustom = null, purposeIds = [], featureIds = [], specialPurpose=null, specialFeature=null } = {}) => {
```

Comment:
Are those default values for specialPurpose and specialFeatures are necessary? If they are necessary, they should be boolean I guess.